docs/recipes/installTips.md

```diff
@@ -364,6 +363,254 @@ GUI will open!
+```
+Then finish installation:
+
+`sudo apt install nvidia-cuda-toolkit gcc-9`
+
+Then check:
+
+`nvcc --version`
+
+If error messages, read them carefully as they often tell you how to fix it, or what to google :D
+
+Now you can see CUDA, DRIVER, GPU(s):
+
+`nvidia-smi`
+
+output:
+
+```python
+$ nvidia-smi
+Sun Jan  1 10:14:06 2023       
++-----------------------------------------------------------------------------+
+| NVIDIA-SMI 510.108.03   Driver Version: 510.108.03   CUDA Version: 11.6     |
+|-------------------------------+----------------------+----------------------+
+| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
+| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
+|                               |                      |               MIG M. |
+|===============================+======================+======================|
+|   0  NVIDIA GeForce ...  Off  | 00000000:01:00.0 Off |                  N/A |
+|  0%   49C    P8    17W / 350W |      5MiB / 24576MiB |      0%      Default |
+|                               |                      |                  N/A |
++-------------------------------+----------------------+----------------------+
+                                                                               
++-----------------------------------------------------------------------------+
+| Processes:                                                                  |
+|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
+|        ID   ID                                                   Usage      |
+|=============================================================================|
+|    0   N/A  N/A      1275      G   /usr/lib/xorg/Xorg                  4MiB |
++-----------------------------------------------------------------------------+
+
+```
+
+Sweet!
+
+
+### Next, Anaconda!
+
+Click here to get the ubuntu/linux package: https://www.anaconda.com/products/individual#linux
+
+this downloads a file, save it somewhere. (~/Downloads is OK)
+
+then enter the folder:
+
+`cd Downloads`
+
+and run:
+
+`bash Anaconda3-2021.05-Linux-x86_64.sh`
+
+and you get:
+```python
+Welcome to Anaconda3 2021.05
+
+In order to continue the installation process, please review the license
+agreement.
+Please, press ENTER to continue
+>>>
+```
+
+Follow prompts, so many prompts!
+
+### Next, DeepLabCut!
+
+Given this is a totally fresh install, here are a few things that also needed: 
+
+`sudo apt install libcanberra-gtk-module libcanberra-gtk3-module`
+
+DeepLabCut can be installed via:
+* Docker: We strongly recommend for Ubuntu users to use Docker (https://hub.docker.com/r/deeplabcut/deeplabcut) - it's a much more reproducible environment.
+* A conda file provided via the DeepLabCut website. This may be a bit more advanced in the development
+* A simple installation through pip: either full GUI (recommeneded) or without GUI (called: DLCLite)
```

Comment:
Once conda, cuda and all other "system" packages are installed, I'm not sure we should continue with the description on how to run DLC (installing via Conda or PIP, with or without GUI etc.).

I would prefer linking to the existing documentation, instead of duplicating it. The reasoning being that if anything changes in DLCs installation process, we'll need to come modify these files as well as the original ones (which is unlikely to happen), hence we would have deprecated installation docs here.

If we do decide to keep this detailed installation information, I would try to make it a bit more condensed, as noted in the comments below.